launch.py

```diff
@@ -12,6 +12,7 @@
 parser.add_argument('-l', '--label', default='INBOX')
 parser.add_argument('-p', '--prefix', default='\uf0e0')
 parser.add_argument('-c', '--color', default='#e06c75')
+parser.add_argument('-t', '--text-color', default='#ffffff')
```

Comment:
I prefer to see only icon (prefix) color change as indicator of new mail. So now, to keep text color for new mail unchanged I need to set it up explicitly. Not gonna merge this one.

Reply:
I understand that you don't use text as a "notificator" but this would add a new feature to the tool in case someone wants to set some text (with some color)

Reply:
Ok, but in case someone doesn't want to set any text color it's white now. This is wrong assumption that someone uses white as default text color for Polybar.

Reply:
Oh ok I see.. What about --custom-text and --text-color flags? If they don't pass any custom text its only the prefix (as you use) and if they do, they can choose the text and we can set custom_text default to ''.

Reply:
I don't get it. What is `--custom--text` about? Give an example please.

Reply:
It was to give an option for the user to set a custom text besides the notification color like. Giving it a second thought, I think the minimalist approach of only prefix or prefix + number works better.